sig/src/Decon2DResponse.cxx

```diff
@@ -0,0 +1,89 @@
+#include "WireCellSig/Decon2DResponse.h"
+#include "WireCellUtil/NamedFactory.h"
+#include "WireCellUtil/String.h"
+#include "WireCellUtil/Array.h"
+
+#include "WireCellAux/SimpleTensorSet.h"
+#include "WireCellAux/SimpleTensor.h"
+#include "WireCellAux/Util.h"
+#include "WireCellIface/ITensorSet.h"
+#include "WireCellUtil/Exceptions.h"
+
+WIRECELL_FACTORY(Decon2DResponse, WireCell::Sig::Decon2DResponse,
+                 WireCell::ITensorSetFilter, WireCell::IConfigurable)
+
+using namespace WireCell;
+
+Sig::Decon2DResponse::Decon2DResponse() : l(Log::logger("pytorch")) {}
+
+Configuration Sig::Decon2DResponse::default_configuration() const
+{
+    Configuration cfg;
+
+    return cfg;
+}
+
+void Sig::Decon2DResponse::configure(const WireCell::Configuration &cfg)
+{
+    m_cfg = cfg;
+}
+
+namespace
+{
+std::string dump(const ITensorSet::pointer &itens) {
```

Comment:
Maybe can go to `Util.h`?

Reply:
Generalized for n-dim and put in `Util`.